stdlib/public/Concurrency/AsyncPrefixWhileSequence.swift

```diff
@@ -14,6 +14,29 @@ import Swift
 
 @available(SwiftStdlib 5.5, *)
 extension AsyncSequence {
+  /// Returns an asynchronous sequence, containing the initial, consecutive
+  /// elements of the base sequence that satisfy the given predicate.
+  ///
+  /// Use `prefix(while:)` to produce values while elements from the base
+  /// sequence meet a condition you specify. The modified sequence ends when
+  /// the predicate closure returns `false`.
+  ///
+  /// In this example, an asynchronous sequence called `Counter` produces `Int`
+  /// values from `1` to `10`. The `prefix(while:)` function causes the modified
+  /// sequence to pass along values so long as they aren’t divisible by `2` and
+  /// `3`. Upon reaching `6`, the sequence ends:
+  ///
+  ///     let stream = Counter(howHigh: 10)
+  ///         .prefix { $0 % 2 != 0 || $0 % 3 != 0 }
+  ///     for try await number in stream {
+  ///         print("\(number) ", terminator: " ")
+  ///     }
+  ///     // prints "1  2  3  4  5"
+  /// - Parameter predicate: A closure that takes an element as a parameter and
```

Comment:
Earlier example code includes a blank line before the parameter section.

Reply:
Added.